jira_agile_metrics/calculators/cycletime_test.py

```diff
@@ -107,6 +107,22 @@ def jira_with_skipped_columns(custom_fields):
                 Change("2018-01-04 01:01:01", [("status", "Next", "Done",), ("resolution", None, "done")]), # skipping columns Build and Test
             ],
         ),
+        Issue("A-11",
+            summary="More Gaps",
+            issuetype=Value("Story", "story"),
+            status=Value("Done", "done"),
+            resolution=Value("Done", "Done"),
+            resolutiondate="2018-01-04 01:01:01",
+            created="2018-01-01 01:01:01",
+            customfield_001="Team 1",
+            customfield_002=Value(None, 10),
+            customfield_003=Value(None, []),
+            customfield_100=None,
+            changes=[
+                Change("2018-01-02 01:05:01", [("status", "Backlog", "Build",)]),
+                Change("2018-01-04 01:01:01", [("status", "Build", "Done",), ("resolution", None, "done")]), # skipping columns Build and Test
+            ],
+        ),
```

Comment:
I added the case you described @rafalotufo . All tests pass without changes to any other code.